org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/commands/OrganizeImportsCommandTest.java

```diff
@@ -349,6 +397,9 @@ public void testOrganizeImportsInProject() throws CoreException, BadLocationExce
 
 	private String getOrganizeImportResult(ICompilationUnit cu, WorkspaceEdit we) throws BadLocationException, CoreException {
 		List<TextEdit> change = we.getChanges().get(JDTUtils.toURI(cu));
+		if (change == null) {

```

Comment:
What is the use case here?

Reply:
'change' is null if there aren't any actions.